documentation/howto/maven_basics.adoc

```diff
@@ -0,0 +1,183 @@
+= Basic Maven Configuration
+:page-layout: howto
+:page-tab: docs
+:page-status: green
+:experimental:
+:imagesdir: ./images
+:toc:
+
+In the context of application development, Maven provides a standardized build system for projects. One of the main benefits of using Maven with your project is that it facilitates fetching dependencies from one or more repositories. This article serves as an introduction to using Maven with the IDE and contains instructions for the following tasks:
```

Comment:
I'm afraid I don't quite know the difference between the functionality offered by Maven for Eclipse-based IDEs vs general IDEs. Any chance you can help clarify this, @fbricon? This introduction is from the 7.1 docs for Maven and seemed valid still to me.